README.md

```diff
@@ -4,11 +4,11 @@
 [![Linting](https://img.shields.io/badge/PEP8%20by-Hound%20CI-a873d1.svg)](https://houndci.com)
 
 ## Documentation
-This repository is a part of the Shotgun Pipeline Toolkit.
+This repository is a part of the ShotGrid Pipeline Toolkit.
 
 - For more information about this app and for release notes, *see the wiki section*.
 - For general information and documentation, click here: https://support.shotgunsoftware.com/entries/95441257
-- For information about Shotgun in general, click here: http://www.shotgunsoftware.com/toolkit
+- For information about ShotGrid in general, click here: http://www.shotgunsoftware.com/toolkit
```

Comment:
Do you want to change the link here as well or leave it until later?

Reply:
Fixed. Thank you. I forgot to update to your version of the script. Thanks!